src/tsconfig.spec.json

```diff
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
+    "sourceMap": true,
+    "declaration": false,
+    "moduleResolution": "node",
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "lib": [
+      "es2016"
+    ],
+    "outDir": "../out-tsc/spec",
+    "module": "commonjs",
+    "target": "es5",
```

Comment:
Originally this was `es6` - but had to make some changes to get Phantom to work.  Not sure what impact this will have ... maybe bad stack traces when running the specs?  Not sure.